api/documentation/controller.py

```diff
@@ -738,6 +760,110 @@ def addSchemes(self):
             }
         )
 
+        self.addComponent(
+            'schema', 'GenrePropsSchema', 'object',
+            {
+                'name': {'type': 'string'},
+                'parents': {
+                    'type': 'array',
+                    'items': {'type': 'string'}
+                },
+                'subgenres': {
+                    'type': 'array',
+                    'items': {'type': 'string'}
+                }
+            }
+        )
+
+        self.addComponent(
+            'schema', 'GenresSchema', 'object',
+            {
+                'Fiction': {'$ref': '#/components/schemas/GenrePropsSchema'},
+                'Nonfiction': {'$ref': '#/components/schemas/GenrePropsSchema'}
+            }
+        )
+
+        self.addComponent(
+            'schema', 'CollectionsGetSchema', 'object',
+            {
+                'collections': {
+                    'type': 'object',
+                    'properties': {
+                        'id': {'type': 'string'},
+                        'name': {'type': 'string'},
+                        'protocol': {
+                            'type': 'string',
+                            'description': 'A protocol name string'
+                        },
+                        'parent_id': {'type': 'string'}
+                    }
+                },
+                'protocols': {
+                    'type': 'object',
+                    'properties': {
+                        'name': {'type': 'string'},
+                        'label': {'type': 'string'},
+                        'description': {'type': 'string'},
+                        'settings': {
+                            'type': 'array',
+                            'items': {
+                                'type': 'object',
+                                'description': 'Mirror integration settings'
+                            }
+                        }
+                    }
+                }
+            }
+        )
+
+        self.addComponent(
+            'schema', 'CollectionsLibraryRegistrationPost', 'object',
+            {
+                'collection_id': {'type': 'string'},
```

Comment:
Is this a single string? Or does it accept an array? If it's just a single `collection_id` we should update the description to state that we can only add collections to a library one at a time.